src/main/scala/firrtl/annotations/AnnotationUtils.scala

```diff
@@ -3,9 +3,15 @@
 package firrtl
 package annotations
 
+import net.jcazevedo.moultingyaml._
+import firrtl.annotations.AnnotationYamlProtocol._
+
 import firrtl.ir._
 
 object AnnotationUtils {
+  def toYaml(a: Annotation): String = a.toYaml.prettyPrint
+  def fromYaml(s: String): Annotation = s.parseYaml.convertTo[Annotation]
```

Comment:
With a better serialization library (probably https://github.com/scala/pickling) specifying a transform won't be required. But even now, you don't **have** to use that field, you just can't use `getMyAnnotations` in a given Transform (which kind of makes sense).